### ISSUE TITLE
Request - Wrap title of notes in file explorer

### ISSUE THREAD

**Issue by danieltomasz** [nikbrunner/obsidian-deep-work-theme#4]
Great theme!

Your theme doesnt wrap name of the note (the vanilla Obsisidian doesn't do it too)
![image](https://user-images.githubusercontent.com/7980381/128242733-2ddb0a26-c2b1-4de1-a102-daa78c413d12.png)
But some  themes  wrap the name (this is Dark Graphite Pie)
![image](https://user-images.githubusercontent.com/7980381/128243436-98e22903-8475-4f82-b725-2724aa31a6d5.png)

Do you consider adding  name wrapping to your theme? If not, I will myself  locally add it as and standalone CSS piece

**Comment by nikbrunner** [comment]
Hi there, :) 
Thank you. Yeah sure I'll look into it. :)

**Comment by danieltomasz** [comment]
I asked on Obsidian discord, this is what Dark Graphite Pie use
```
/* Wrap long nav text and some paddings */
.nav-file-title,
.nav-folder-title {
  white-space: normal;
  width: auto;
}
```

**Comment by nikbrunner** [comment]
Merged with c6faefd5b704313047b7a2d72494f121738c5752 :)

**Comment by danieltomasz** [comment]
Thanks!
